### PR TITLE
added values of kafka broker and kafka topic for transformer cronjob

### PR DESCRIPTION
**Pull request by prerna-egov**

### CODE REVIEW

File: deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml [hunk 1]
```diff
@@ -38,6 +38,13 @@ env: |
     value: "individual-index-v1"
   - name: ES_TRANSFORMER_PI_INDEX
     value: "transform-ifms-pi-index"
+  - name: KAFKA_BROKER
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: kafka-brokers
+  - name: TRANSFORMER_PI_KAFKA_TOPIC
```
Comment (shailesh-egov) [inline deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml:58]: @prerna-egov please add conditional values, we can not update that according to enviornment.

**Review by shailesh-egov — COMMENTED** [review]
Remove all hard coded values.

File: deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml (hunk 1 above)
Comment (prerna-egov) [reply deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml:58]: @shailesh-egov made the changes. please check

Comment (shailesh-egov) [reply deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml:58]: @prerna-egov your if is not in the correct place. Please check the given example and change it accordingly
https://github.com/odisha-muktasoft/DIGIT-DevOps/blob/mukta-1.22-changes/deploy-as-code/helm/charts/digit-works/utilities/ifms-cronjob-pa/values.yaml#L39

Comment (prerna-egov) [reply deploy-as-code/helm/charts/digit-works/utilities/transformer-cronjob/values.yaml:58]: @shailesh-egov I took reference from here https://github.com/egovernments/DIGIT-DevOps/blob/unified-env/deploy-as-code/helm/charts/common/values.yaml#L41

**Review by shailesh-egov — APPROVED** [review]

**Review by egov-arindam — APPROVED** [review]

**Review by egov-arindam — APPROVED** [review]

**Pull request merged**